cbins13/dotTheory---FE
# FE-864: The Finer Things

## Related Tickets
- [FE-864](https://codeandtheory.atlassian.net/browse/FE-864)

## Description
- This pull request is created for the module FE-403, where the checklist includes:

1. In Confluence, write a comparison document about the CSS Design Patterns included above, but instead of simply listing down the similarities/differences, write about what you like and don't like about each one. And as a summary, indicate which among the patterns you prefer the most - Document is found [here](https://codeandtheory.atlassian.net/wiki/spaces/~807204863/pages/3626631538?atlOrigin=eyJpIjoiOTJlOTRjOTQwMTY0NDFmNzk3NGJhNDkyOGI3ODJmZGUiLCJwIjoiYyJ9)
2. Refactor your project's codebase to use the preferred methodology you've indicated above
3. Refactor your project's codebase to use CSS Variables instead of Sass variables

## Validation
- Install Sass, and Code Editor of your choice (VSCode, Atom, Sublime)
- Open the index.html on Live Server via VSCode to view the detail page, or open the index.html on a browser
- Open terminal and navigate to the the root directory of the project (html-css-project)
- Enter the command sass --watch scss:css :compressed 
- Under the css folder, open the application.css file to view the output css, or
- Under the browser, press the f12 key to view an element and see the implementation of the SMACSS CSS design pattern
